plugins/logitech-hidpp/fu-logitech-hidpp-device.c

```diff
@@ -32,6 +33,15 @@
  */
 #define FU_LOGITECH_HIDPP_DEVICE_FLAG_REBIND_ATTACH (1 << 2)
 
+/**
+ * FU_LOGITECH_HIDPP_DEVICE_FLAG_ADD_RADIO:
+ *
+ * The device should add a softdevice (index 0x5), typically a radio.
+ *
+ * Since: 1.7.0
+ */
+#define FU_LOGITECH_HIDPP_DEVICE_FLAG_ADD_RADIO (1 << 5)
```

Comment:
why the jump from 2 to 5?

Reply:
I'm chunking a mega commit -- the 3 and 4 will appear soon

Reply:
ah okay.